src/legacy/core_plugins/kibana/public/discover/doc_table/lib/get_sort.js

```diff
@@ -61,7 +58,11 @@ export function getSort(sort, indexPattern, defaultSortOrder = 'desc') {
   }
 }
 
-getSort.array = function (sort, indexPattern, defaultSortOrder) {
-  return getSort(sort, indexPattern, defaultSortOrder).map((sortPair) => _(sortPair).pairs().pop());
+/*
+ * Take an array of elasticsearch sort objects and return an array of arrays.
+ * Basically the opposite of `getSort`.
+ */
+getSort.array = function (sort) {
```

Comment:
The purpose of this method seems to have gotten confused over time (quite possibly my own doing). As best as I can tell, it was originally meant to do the reverse of `getSort`: take an elasticsearch sort object and turn it into an array that Discover can consume. So I fixed it to do that and removed some extraneous usages in Discover where it was being used as a glorified way to wrap an array with another array.